### ISSUE TITLE
Document what happens when a validator is specified both ways

### ISSUE THREAD

**Issue by rgabbard** [python-attrs/attrs#314]
As far as I can tell, the documentation doesn't specify what happens if you specify a validator as a keyword argument to `attrib` and also using a decorator.   From testing, it appears they both run, but it is not clear to me if this is guaranteed (or if the order is guaranteed).

```
@attrs
class Foo:
   field = attriib(validator=instance_of(str))

   @field.validator
   def _validate_field(...):
     ...
```

**Comment by hynek** [comment]
FTR, it is guaranteed that both are ran.

Documenting the decorators is really painful because they aren't on a public class.  I will have to write a narrative chapter about this stuff. :|

**Comment by hynek** [comment]
fixed in #370
